plugins/modules/lvg.py

```diff
@@ -418,89 +369,40 @@ def make_vg(module, vg_name, vg_state):
     global result
 
     pvs = module.params['pvs']
-
-    cmd = "mkvg "
+    opt = build_vg_opts(module)
 
     vg_type_opt = {
```

Comment:
This is not consistent, are we supposed to add small here? I don't see "small" in the dict.